bin/templates/ext/yarp/node.c.erb

```diff
@@ -11,9 +11,7 @@ extern VALUE rb_cYARPLocation;
 
 static VALUE
 location_new(yp_parser_t *parser, const char *start, const char *end) {
-  VALUE real_start = start ? LONG2FIX(start - parser->start) : Qnil;
-  VALUE real_end = end ? LONG2FIX(end - parser->start) : Qnil;
-  VALUE argv[] = { real_start, real_end };
+  VALUE argv[] = { LONG2FIX(start - parser->start), LONG2FIX(end - parser->start) };
```

Comment:
It would be good to `assert(start); assert(end);` here to make sure we don't pass the value of `-parser->start` as an "offset" to Ruby, like done in https://github.com/Shopify/yarp/blob/e2972023e6b7d06de7caaec6f3858e697a988a1b/bin/templates/src/serialize.c.erb#L22-L23